src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+
+![CUPS Status Window](/images/printers/cups-settings.png)
+
+CUPS allows you to manage printers using Terminal commands. See the [CUPS documentation](https://www.cups.org/doc/admin.html) for more information.
+
+## Adding a Printer
+
+Most printers will be automatically added to the computer. If a printer is not automatically added, you can manually add printers in `Settings` ➞ `Printers`.
+
+1. Press the `Super` key and type the word Printers.
+
+2. Choose the `Printers` application in the search box.
+
+3. In the `Printers Application`, click the `Add a Printer...` button and a box will pop up with different options.
+
+    ![Click Add a Printer](/images/printers/add-a-printer.png)
+
+4. Wait a few seconds for printers to appear in the Device List.
+
+5. Select your printer and click `Add`.
+
+    ![Click Add](/images/printers/click-add.png)
+
+## HP Printers
+
+HP printers are supported with the hplip package, which is installed by default in Pop!\_OS.
+
+```bash
+sudo apt install hplip
+```
+
+### HP Device Manager (GUI)
+
+If you would like to use a guided GUI application from HP, you will need to install a python dependency:
+
+```bash
+sudo apt install python3-pyqt5
+```
+
+Then run `hp-setup` to start the HP Device Manager.
+
+![HP Device Manager](/images/printers/hp-setup.png)
+
+## Brother Printers
+
+Brother provides a driver installation tool for Linux users. Install the appropriate driver for your Brother printer by downloading this tool and running the installer with your printer model appended to the command.
+
+1. Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to download Brother's Driver Install Tool for Linux while searching for the appropriate printer.
+
+2. Click the `Click here to download the tool` link on the Brother download page.
+
+3. Select Linux (deb) for your "OS Version", then click `OK`.
+
+4. Click `Agree to the EULA and Download`.
+
+5. Choose the `Save File` option when prompted.
+
+6. Press `Super` + `T` to launch the Terminal application.
+
+7. Change directory to where you downloaded the driver (usually the Downloads directory). Unzip the file using this command:
+
+    ```bash
+    cd Downloads
+    gunzip linux-brprinter-installer-*.gz
+    ```
+
+8. Run the installer by typing the unzipped installer name into the terminal.
+
+    > **Note**: Your installer version may differ from this guide. Type the first portion of the installer name as shown below, and then hit <kbd>TAB</kbd> to complete the installer name. Place your exact printer model where we wrote PRINTERNAME below.
+
+   ```
+   sudo bash linux-brprinter-installer PRINTERNAME
+   ```
+  
+    > **Note**: If prompted for a "DeviceURI", you can find that by opening up Settings > Printers > Additional Printer Settings, then right click your printer and click Properties. In the resulting window, you'll be able to find your Device URI, as shown in the screenshot below.
```

Comment:
> If prompted for a "DeviceURI",

A comma is technically supposed to be inside of quotation marks, regardless of if the quotation includes the comma. For that reason, I would recommend just using inline code backticks instead.

> If prompted for a `DeviceURI`,

> you can find that by opening up Settings > Printers > Additional Printer Settings, then right click your printer and click Properties.

"opening up" and "right click" are not the same tense. The second one should be "right clicking" (and "click Properties" should be "clicking Properties.")